lib/did-io.js

```diff
@@ -128,7 +128,8 @@ api.use = function(name, injectable) {
         globalName = 'jsonldjs';
         break;
       case 'uuid':
-        requireName = 'node-uuid';
+        // NOTE: the `uuid` module does not work in the browser
```

Comment:
If `uuid` doesn't work in the browser and we're removing `node-uuid` from the package dependencies, how will the browser version work properly? Do we just throw an error if not manually configured?